loadtest/beats-yaml-generator.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+stream = os.popen('kubectl get pods -o wide -n logstash')
+output = stream.read()
+output = output.split("\n")
+result = ""
+
+i = 0
+for o in output:
+    if i > 0 and o != "":
+        s = o.split()
+        if i >= 2:
+            result += ", "
+        result += '"%s:5044"' % s[5]
+    i += 1
+
+beats = ""
```

Comment:
Do we need some sort of error check here if it didn't parse out the Logstash pod.

Reply:
I added a check to avoid adding pods with `<none>` as an IP, for any exceptions I think there isn't anything to handle, having it bubble up and get printed might be our best bet. The output for when there is no pods or no namespace are only one line and get skipped over as a result, so are handle.